docs/mine/lotus/miner-wallets.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Lotus Miner: Wallets'
+description: 'Miners make use of wallets managed by the Lotus Node to both send and receive FIL during the normal course of operation (sending proofs, collecting deal payments etc.). This guides explains how to configure different wallets so the miner can use them for different operations.'
+breadcrumb: 'Miner Wallets'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }}
+
+As mentioned during the [init phase](miner-setup.md) of the Lotus Miner, it is required to provide an owner address with some funds that the miner will use for all the operations needed (or it will use the [Lotus Node default wallet](../../get-started/lotus/send-and-receive-fil.md)). This address will be used by the miner for all the operations, but in many cases, it makes sense to use dedicated wallets:
+
+However, different components of the miners can be configured to use different wallets:
+
+[[TOC]]
+
+## The owner address
+
+The owner address is used to receive payments and to pay the fees for any miner-actor triggered changes like changing its information on changing addresses (TODO: actually list exactly what it is used for).
+
+You should have specified it during the `lotus-miner init` phase (otherwise, the default Lotus wallet will be used).
+
+This wallet should have enough balance to pay for actor-actions like changing the information.
+
+Given the high value of this wallet, we recommend taking special measures to keep it safe (i.e. keep it in cold storage as long as there are no miner actions to take and the address is not re-used for something else).
+
+## The worker address
+
+The worker address is used to sign blocks and submit proofs associated with this miner. It must be a BLS address and can be specified during `init` with the `--worker` flag.
```

Comment:
There seems to be no other way to specify a worker address after initialization? In that case, the setup instructions need to be clearer about that so that this step is not missed if needed.

Reply:
Also should explain why someone would want a worker address. I am assuming the owner address mostly receives payments.